examples/golang/waku.go

```diff
@@ -446,6 +451,20 @@ func (self *WakuNode) WakuListenAddresses() (string, error) {
 	return "", errors.New(errMsg)
 }
 
+func (self *WakuNode) WakuGetMyENR() (string, error) {
+	var resp = C.allocResp()
+	defer C.freeResp(resp)
+	C.cGoWakuGetMyENR(self.ctx, resp)
+
+	if C.getRet(resp) == C.RET_OK {
+		var listenAddresses = C.GoStringN(C.getMyCharPtr(resp), C.int(C.getMyCharLen(resp)))
+		return listenAddresses, nil
```

Comment:
Shouldn't we change the variable name to `enr` or something like that? looks like we kept the same variable name after copy-pasting from `WakuListenAddresses()`

Reply:
ooh @NagyZoltanPeter added the same comment a few minutes before I did lol. Nice!

Reply:
Thank you so much guys ! I'll change that